src/index.ts

```diff
@@ -85,8 +85,9 @@ export default async function app(
 
   app.register(helmet);
   app.register(cors, {
-    origin: isProd ? /daily\.dev$/ : true,
+    origin: isProd ? /^(?:https:\/\/)?(?:[\w-]+\.)*daily\.dev$/ : true,
```

Comment:
before it would allow `adaily.dev`

Reply:
Good catch, otherwise we'd have to buy *daily.dev domains there are 💸